ui/components/institutional/wrong-network-notification/wrong-network-notification.tsx

```diff
@@ -14,12 +14,19 @@ import { useI18nContext } from '../../../hooks/useI18nContext';
 import { getProviderConfig } from '../../../ducks/metamask/metamask';
 import { Icon, IconName, IconSize, Box, Text } from '../../component-library';
 
-const WrongNetworkNotification = () => {
+type ProviderConfig = {
+  nickname?: string;
+  type: string;
+};
```

Comment:
Did you consider using the `ProviderConfig` definition from `@metamask/network-controller` instead of defining a new type?